components/markdown/src/markdown.rs

```diff
@@ -679,17 +684,15 @@ pub fn markdown_to_html(
                         event
                     });
                 }
-                Event::Html(text) => {
-                    if !has_summary && MORE_DIVIDER_RE.is_match(&text) {
-                        has_summary = true;
-                        events.push(Event::Html(CONTINUE_READING.into()));
-                        continue;
-                    }
-                    if !contains_shortcode(text.as_ref()) {
-                        events.push(Event::Html(text));
-                        continue;
-                    }
-
+                Event::Html(text) | Event::InlineHtml(text)
```

Comment:
I believe it might better to remove the InlineHtml from there for now so you don't get broken HTML

Reply:
That's fair; I'll make that change but leave in the test that shows that there's no summary when this happens.